var/spack/repos/builtin/packages/cosp2/package.py

```diff
@@ -0,0 +1,73 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import shutil
+
+
+class Cosp2(MakefilePackage):
+    """ Proxy Application. CoSP2 represents a sparse linear algebra
+        parallel algorithm for calculating the density matrix in electronic
+        tructure theory. The algorithm is based on a recursive second-order
+        Fermi-Operator expansion method (SP2) and is tailored for density
+        functional based tight-binding calculations of non-metallic systems
+        tags = proxy-app
```

Comment:
If you run `spack info` you'll see that the formatting is a little weird because newlines aren't preserved. You'll want a period after systems. Or just leave out `tags = proxy-app` since it isn't really a description. I've never seen any other package in Spack that does this.